### ISSUE TITLE
button and slider broken on touchscreens

### ISSUE THREAD

**Issue by ulno** [node-red/node-red-dashboard#523]
### What are the steps to reproduce?
- Create a slider or button dashboard node, connect to debug
- Deploy
- Access from browser on an Android device (tested Chromium, Firefox, Fennec, Fenix, Brave, Fully) or Raspberry Pi with touch screen (tested with Chromium)

### What happens?
- Move slider with finger -> no result will be visible in debug (only clicking next to the slider button will trigger message)
- Touch a button and release screen -> the button will stay selected and on some android devices sporadically trigger the pressed message again after a while (due to some screensaver and sleep related events on Android)

### What do you expect to happen?
Same as using with mouse (using a real mouse on the Android device works as expected):
- Slider sends values when moved
- Button deselects when released

### Please tell us about your environment:

- [x] Node-RED-Dashboard version: 2.15.5
- [x] Node-RED version: 0.20.7
- [x] node.js version: v10.16.2
- [x] npm version: 6.9.0
- [x] Platform/OS: Devuan Ascii (also tested on Raspbian stretch, not yet buster)
- [x] Browser: latest Chromium, Firefox, Fennec, Fenix, Brave, Fully Android versions (default Chromium on current Raspbian)

A quick search of the dashboard source code yields that ng-touchstart and ng-touchend are never used, only ng-click. I assume, but stand easily corrected, that using these in addition or instead of click with ng-mousedown and ng-mouseup would solve the issue.

Building my own button in a template node works if used the following way:
```html
<script>
this.scope.on = {payload: 1};
this.scope.off = {payload: 0};
</script>
<md-button
   ng-touchstart="send(on)"
   ng-mousedown="send(on)"
   ng-touchend="send(off)"
   ng-mouseup="send(off)">
       Agua
</md-button>
```

(It does though also not work on one of my older Android devices using the Fully browser - on all other devices that works - maybe some kind of older web engine on that device?)
This is code from the second related discussion below.

I assume that this effects all touchscreen uses of the node-red dashboard and other gui elements that use interaction. Maybe some other people can comment here if this hypotheses is true.

Related discussion:
- https://discourse.nodered.org/t/slider-issue-on-android-chrome-browser/13302/7
- https://discourse.nodered.org/t/momentary-button-on-dashboard/4906/5

Issue #503 seems also slightly related.

**Comment by dceejay** [comment]
Duplicate of #519

**Comment by dceejay** [comment]
now fixed in v2.22.1
